pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[tool.black]
+line-length = 140
```

Comment:
FYI, I tried it and line-length setting from pyproject.toml is also used in pre-commit which is great